android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/MetadataResolver.java

```diff
@@ -236,6 +249,17 @@ public void onErrorResponse(VolleyError volleyError) {
     );
   }
 
+  public static boolean checkIfMetadataContainsSortingScores(Collection<UrlMetadata> urlMetadataCollection) {
```

Comment:
@dinhviethoa this seems hacky and unnecessary. Can you just return whether or not the score is on in the JSON response? I don't see why we have to implement it this way.